submissions/Eugene-CG/memory-pair-game/index.js

```diff
@@ -0,0 +1,92 @@
+const cardsList = document.querySelector(".cards__list");
+let clicksStorage = [];
+
+const startGame = () => {
+  let cards = [
+    {
+      imgId: "fool",
+      src: "fool.jpg",
+    },
+    {
+      imgId: "lovers",
+      src: "lovers.jpg",
+    },
+    {
+      imgId: "magician",
+      src: "magician.jpg",
+    },
+    {
+      imgId: "priestess",
+      src: "priestess.jpg",
+    },
+    {
+      imgId: "star",
+      src: "star.jpg",
+    },
+    {
+      imgId: "sun",
+      src: "sun.jpg",
+    },
+    {
+      imgId: "hanged",
+      src: "hanged.jpg",
+    },
+    {
+      imgId: "justice",
+      src: "justice.jpg",
+    },
+  ];
+  cards = shuffleCards([...cards, ...cards]);
+
+  let cardListInnerHtml = ``;
+  let i = 0;
+  cards.forEach((obj) => {
+    cardListInnerHtml += createCard(obj, i++);
+  });
+  cardsList.innerHTML = cardListInnerHtml;
+};
+const shuffleCards = (cards) => cards.sort(() => Math.random() - 0.5);
+let arr = [];
+const createCard = ({ imgId, src }, i) => {
+  return `
+    <li class="card__container">
+          <div class="card card-back">
+              <img src="./img/${src}" id="${imgId}${i}">
+          </div>
+      </li>`;
+};
+const flipCard = (target) => {
+  if (target.closest(".card")) {
```

Comment:
I would move this into listener so `flipCard` will just flip cards. Also, it's a good example of unnecessary nested logic - why do you call function for cases in which it will do nothing? Better not call it all.